migrations/4_batch_exchange_viewer.js

```diff
@@ -0,0 +1,7 @@
+const BatchExchangeViewer = artifacts.require("./BatchExchangeViewer.sol")
+const BatchExchange = artifacts.require("./BatchExchange.sol")
```

Comment:
artifacts.require("BatchExchange") should be sufficient.

Later on, we also might want to use the same migration structure, which we have for the other projects, just so that it be reused.